packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -414,20 +414,22 @@ export function SelectWalletModal({
                         isMobileDevice={isMobile}
                         renderAvatar={renderAvatar}
                       />
-                      <hr className="horizontal-separator" />
                       {!isMobile && (
-                        <WalletButton
-                          wallet={Wallet.MAGICEDEN}
-                          subtitle="Coming soon on mobile browsing"
-                          onConnect={onConnectMagicEdenWallet}
-                          icon={MagicEdenWalletIcon}
-                          setErrorMessage={setErrorMessage}
-                          isDisabled={isMobile}
-                          isMobileDevice={isMobile}
-                          renderAvatar={renderAvatar}
-                        />
+                        <>
+                          <hr className="horizontal-separator" />
+                          <WalletButton
```

Comment:
I saw we have multiple `!isMobile` conditions here, can we combine them into one?

Reply:
Okay, combined them into one. Originally I was thinking of keeping it separate per wallet but this works too.